src/shared/env/env.contants.ts

```diff
@@ -0,0 +1,14 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export const ENV = Object.freeze({
+  PUBLIC_ACCESS_KEY: process.env.PUBLIC_ACCESS_KEY,
+  PRIVATE_ACCESS_KEY: process.env.PRIVATE_ACCESS_KEY,
```

Comment:
In AWS for higher envs it's better to manage access to AWS services through IAM service roles.

Reply:
if you're going to use EC2 for deployment, just add necessary permission to EC2 instance profile role.

Reply:
roger that